projects/israel/pelias.json

```diff
@@ -0,0 +1,93 @@
+{
+  "logger": {
+    "level": "debug",
+    "timestamp": false
+  },
+  "esclient": {
+    "apiVersion": "7.5",
+    "hosts": [
+      {
+        "host": "elasticsearch"
+      }
+    ]
+  },
+  "elasticsearch": {
+    "settings": {
+      "index": {
+        "refresh_interval": "10s",
+        "number_of_replicas": "0",
+        "number_of_shards": "1"
+      }
+    }
+  },
+  "api": {
+    "services": {
+      "pip": {
+        "url": "http://pip:4200"
+      },
+      "libpostal": {
+        "url": "http://libpostal:4400"
+      },
+      "placeholder": {
+        "url": "http://placeholder:4100"
+      },
+      "interpolation": {
+        "url": "http://interpolation:4300"
+      }
+    },
+    "defaultParameters": {
+      "focus.point.lat": 32.055145,
+      "focus.point.lon": 34.756369
+    }
```

Comment:
It's probably better to remove this section, this will always bias results towards Tel Aviv when the user doesn't specify a `focus.point` in the query string. Leaving it blank will better serve users in Jerusalem, etc.

Reply:
Thank you for your feedback. I've removed it.